ui/components/AutoComplete.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
+import * as React from "react";
+import Downshift from "downshift";
+
+
+export const AutoComplete = ({items}) => {
+    return (
+      <div>
+        <Downshift
+          onChange={(selection) => alert(selection ? `You selected ${selection.value}` : "Selection Cleared")}

```

Comment:
Here instead of an alert, you need to pass the `handleSelectionChange` function prop as mentioned above.